.gitattributes

```diff
@@ -0,0 +1,3 @@
+data/*.tar.gz filter=lfs diff=lfs merge=lfs -text
+data/*.tgz filter=lfs diff=lfs merge=lfs -text
+data/*.zip filter=lfs diff=lfs merge=lfs -text
```

Comment:
I'm still not clear on when LFS is required vs not required in local dev. Does this require interacting with LFS for local dev in anyway?

Reply:
Good question 🤔 I think when you change things matching the patterns, you'll have to. But they're currently only used in CI, so in most cases I don't think so. But I'll try this out and otherwise open a follow-up PR updating docs around this. (An external contributor wants to address the serialization bug, so merging this will help set them up easier.)